server/protocols/webadmin-integration-test/memory-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/memory/MemoryForwardIntegrationTest.java

```diff
@@ -19,22 +19,24 @@
 
 package org.apache.james.webadmin.integration.memory;
 
-import java.io.IOException;
-
 import org.apache.james.GuiceJamesServer;
-import org.apache.james.MemoryJmapTestRule;
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.MemoryJamesServerMain;
+import org.apache.james.modules.TestJMAPServerModule;
 import org.apache.james.webadmin.WebAdminConfiguration;
 import org.apache.james.webadmin.integration.ForwardIntegrationTest;
-import org.junit.Rule;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-public class MemoryForwardIntegrationTest extends ForwardIntegrationTest {
+class MemoryForwardIntegrationTest extends ForwardIntegrationTest {
 
-    @Rule
-    public MemoryJmapTestRule memoryJmap = new MemoryJmapTestRule();
+    private static final int LIMIT_TO_10_MESSAGES = 10;
```

Comment:
As stated and agreed with @mbaechler before when I was migrating test classes, JUnit5 is allowing us to reduce verbosity of the code, by not declaring necessarily the access modifiers (which we were doing already for classes and  test method declarations). We tend to remove that with global variables too now.

Reply:
Didn't see any changes on this, why resolved? :)

Reply:
I missed understand you, but I had remove all `LIMIT_TO_10_MESSAGES` declarations to use an unique constant in `TestJMAPServerModule`. That constant could be accessed from many places in the code base, so I would like to keep it public as its scope now not only inside the class.